### ISSUE TITLE
Question about training dataset

### ISSUE THREAD

**Issue by Chanfeechen** [LiYingwei/ShapeTextureDebiasedTraining#1]
I am very interesting in your debiased model and trying to reproduce the results. However I only get **65% top1 accuracy** for **ResNet50** model. I also tried the **pre-trained ResNet50** model provided by the repo, and I only get 0.02% top1 accuracy. So I wonder if I am using the wrong dataset. Here are what I tried.

**Dataset**: Stanford CS231N's  [tiny-ImageNet-200](http://cs231n.stanford.edu/tiny-imagenet-200.zip)
**Training with command**: 
`python imagenet.py -a resnet50 --data $DATA --epochs 100 --schedule 30 60 90 --checkpoint $MODEL_DIR --gpu-id 0 --train-batch 64 --lr 0.2 --mixbn --style --multi_grid `
**Evaluate with command**: 
`python imagenet.py -a resnet50 --data $DATA --mixbn --evaluate --load $CKPT`
Thank you :)

**Comment by LiYingwei** [comment]
Thanks for your interest. We use the ImageNet-1k [training set](https://academictorrents.com/details/a306397ccf9c2ead27155983c254227c0fd938e2) and [validation set](https://academictorrents.com/details/5d6d0df7ed81efd49ca99ea4737e0ae5e3a5f2e5) for training and evaluating. 

A minor comment: we set training batch size as 512 and learning rate as 0.2. If you want to have a smaller batch size, try to use a lower learning rate accordingly. Check section 2.1 of [Accurate, Large Minibatch SGD: Training ImageNet in 1 Hour](https://arxiv.org/pdf/1706.02677.pdf) for details.

**Comment by Chanfeechen** [comment]
Thank you very much for the clarification and the comment. It's very helpful! :)
